### ISSUE TITLE
The test vectors location has moved

### ISSUE THREAD

**Issue by est31** [xiph/opus#53]
Minor nit of the README. It says:

```
There is also collection of standard test vectors which are not
included in this package for size reasons but can be obtained from:
https://opus-codec.org/testvectors/opus_testvectors.tar.gz

To run compare the code to these test vectors:

    % curl -O https://opus-codec.org/testvectors/opus_testvectors.tar.gz
    % tar -zxf opus_testvectors.tar.gz
    % ./tests/run_vectors.sh ./ opus_testvectors 48000

```

but `curl -O` only gives us an HTML file that tells us that the file has moved permanently to https://opus-codec.org/static/testvectors/opus_testvectors.tar.gz , and tar fails to decompress this.

To fix this, either the URL should be updated, or `curl -LO` should be suggested which follows redirects.

**Comment by rillian** [comment]
It looks like the download url isn't in the RFC, so we're free to change the url. We could either change the explicit url or `curl -OL`.

**Comment by rillian** [comment]
Oh, you said that already. I think I favour the `-OL` option since it keeps the line short.

**Comment by rillian** [comment]
Fixed in f57c904732b34d5ee11802d59377607684ddb257. Thanks for the report!
